student.rb

```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', permission: true)
+    super(name, age, permission)
```

Comment:
- [ ] Kindly follow the same order for the parameters in the superclass when using the `super` keyword to avoid any argument error, so it will be like:
```ruby
super(age, name: name, parent_permission: parent_permission)
```